dnf/package.py

```diff
@@ -38,6 +38,10 @@
 class Package(hawkey.Package):
     """ Represents a package. #:api """
 
+    # :api
+    DEBUGINFO_SUFFIX = "-debuginfo"
+    DEBUGSOURCE_SUFFIX = "-debugsource"
```

Comment:
Not entirely sure we want to add this as an API or if this is the proper way to mark it.

Reply:
In theory, it's possible for that suffix to be different, but in practice I don't think so.

Reply:
@lukash, looking at constants defined in ``dnf.callback``, I'd say the proper way to mark it as API is to add ``# :api`` at the end of each line. And it makes it clear whether all the constants are part of API or just the first one.

Reply:
Fixed.